spec/system/posts/index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
```

Comment:
**OPTIONAL** - I had some errors on my PC while running the test, until I used the `type: :feature`. I suggest you fix this for all your integration test files 🙏

Reply:
Thank you @mckent05 i will make the changes and request a new review